packages/web_benchmarks/lib/client.dart

```diff
@@ -320,6 +318,7 @@ class LocalBenchmarkServerClient {
   /// DevTools Protocol.
   Future<void> startPerformanceTracing(String? benchmarkName) async {
     _checkNotManualMode();
+    // ignore: deprecated_member_use
```

Comment:
Removed, the rest of the occurrences are in `mock` or _test files, not in user code.

Reply:
```console
$ grep deprecated_member_use * -Irn | grep "_web" | grep "dart:" | cut -d: -f1 | sort | uniq -c | sort -r
      8 shared_preferences/shared_preferences_web/example/integration_test/shared_preferences_web_test.dart
      4 image_picker/image_picker_for_web/example/integration_test/image_picker_for_web_test.dart
      2 webview_flutter/webview_flutter_web/test/web_webview_controller_test.mocks.dart
      2 webview_flutter/webview_flutter_web/test/legacy/webview_flutter_web_test.mocks.dart
      2 google_sign_in/google_sign_in_web/example/integration_test/web_only_test.mocks.dart
      2 google_sign_in/google_sign_in_web/example/integration_test/google_sign_in_web_test.mocks.dart
      2 google_maps_flutter/google_maps_flutter_web/example/integration_test/overlays_test.mocks.dart
      2 google_maps_flutter/google_maps_flutter_web/example/integration_test/google_maps_plugin_test.mocks.dart
      2 google_maps_flutter/google_maps_flutter_web/example/integration_test/google_maps_controller_test.mocks.dart
```

Do we want to remove these from `_test.dart` files as well?

Reply:
Autogenerated mocks are fine; the `ignore`s there are also auto-generated so not something we maintaine.

The two test files there are specific exceptions called out in the new gardener docs. They are annotating places where we have integration tests of our own deprecated methods (integration tests are technically a different bundle, so trip that check instead of `deprecated_member_use_from_same_package`), and we don't track those as tech debt since they are not actually cross-package, and will be removed in the same PR that removes the deprecated APIs themselves. Since we don't want to track them, we don't want them to show up in the manual audit, so we `ignore` them.

Reply:
Ah, fantastic! I totally missed the gardener docs about those two files 😅

I'll mark this conversation as resolved then!